utils_nlp/models/xlnet/utils.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+from matplotlib import pyplot
+from sklearn.metrics import confusion_matrix
+import seaborn as sn
+import numpy as np
+import pandas as pd
+def generate_confusion_matrix(y_true, y_pred,labels):
+    """Function that prints out a graphical representation of confusion matrix using Seaborn Heatmap
+
+    Args:
+        y_true: True labels from dataset
+        y_pred: Predicted labels from the models
+        labels: A list of labels
+
+    """
+    conf_matrix=np.array(confusion_matrix(y_true, y_pred))
```

Comment:
in case you need inspiration, you can take a look at this function: https://github.com/miguelgfierro/pybase/blob/master/plot_base/plot_confusion_matrix.py

Reply:
thanks - will look into that